### PR TITLE
Update for v1.4

### PR DESCRIPTION
**Pull request by lorensr**

### CODE REVIEW

File: ejson/src/ejson-payload-converter.ts [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
-  EncodingType,
-  errorMessage,
-  METADATA_ENCODING_KEY,
-  Payload,
-  PayloadConverterWithEncoding,
-  str,
-  u8,
-} from '@temporalio/common';
+import { EncodingType, METADATA_ENCODING_KEY, Payload, PayloadConverterWithEncoding } from '@temporalio/common';
+import { errorMessage } from '@temporalio/common/lib/type-helpers';
```
Comment (bergundy) [inline ejson/src/ejson-payload-converter.ts:3]: Let's not use this internal API

File: ejson/src/ejson-payload-converter.ts [hunk 2]
```diff
@@ -1,13 +1,7 @@
...
+import { decode, encode } from '@temporalio/common/lib/encoding';
```
Comment (bergundy) [inline ejson/src/ejson-payload-converter.ts:4]: As mentioned on slack, let's use an external lib for this.

**Review by bergundy — COMMENTED** [review]
Look below at the deprecated uses of `u8` and `str`, we should replace those with TextEncoder calls (requires `lib: 'dom'` in `tsconfig.json`).

File: hello-world/src/workflows.ts [hunk 3]
```diff
@@ -9,6 +9,11 @@ const { greet } = proxyActivities<typeof activities>({
 
 /** A workflow that simply calls an activity */
 export async function example(name: string): Promise<string> {
-  return await greet(name);
+  const minuteP = sleep('1m');
```
Comment (bergundy) [inline hello-world/src/workflows.ts:12]: Assuming this wasn’t supposed to be committed

Comment (lorensr) [reply hello-world/src/workflows.ts:12]: thanks!

**Review by vkarpov15 — APPROVED** [review]

**Pull request merged**